test/metabase/cmd/load_and_dump_test.clj

```diff
@@ -24,9 +22,10 @@
     (let [h2-fixture-db-file (abs-path "frontend/test/__runner__/test_db_fixture.db")
           h2-file            (abs-path "/tmp/out.db")
           db-name            "dump-test"]
-      (mt/test-drivers #{:postgres :mysql :h2}
+      (mt/test-drivers #{:mysql :postgres :h2}
         (h2/delete-existing-h2-database-files! h2-file)
-        (binding [mdb.connection/*db-type*   driver/*driver*
+        (binding [setting/*disable-cache* true
+                  mdb.connection/*db-type*   driver/*driver*
```

Comment:
If the Settings cache causes problems, would it make sense to disable it whenever we run the command (as opposed to only in tests)?

Reply:
I don't know enough about the settings caching behavior.  
The spurious errors seemed to be happening only on tests (never happened locally), and generally only in presto. I thought it was something about the synchronization:

```
Table "SETTING" not found; SQL statement:
SELECT "SETTING"."VALUE" FROM "SETTING" WHERE ("KEY" = ? AND "VALUE" > ?) LIMIT 1 [42102-197]
```

failed test: https://app.circleci.com/pipelines/github/metabase/metabase/12278/workflows/236356a5-b246-48fd-9c4e-fdce1999ca7e/jobs/457485

Reply:
Merging it for now, 
If you have the intuition that the problem is still the real code and not the tests and we should takle it now, ping again and I'll investiate it further